src/components/PeopleFilters.tsx

```diff
@@ -1,12 +1,63 @@
+import { useContext } from 'react';
+import cn from 'classnames';
+import { useSearchParams } from 'react-router-dom';
+import { PeopleContext } from '../PeopleContext';
+import { getSearchWith } from '../utils/searchHelper';
+import { SearchLink } from './SearchLink';
+
+const queryCentury = ['16', '17', '18', '19', '20'];
+
 export const PeopleFilters = () => {
+  const { handleQueryChange } = useContext(PeopleContext);
+
+  const [searchParams] = useSearchParams();
+
+  const sex = searchParams.get('sex') || 'all';
+  const query = searchParams.get('query') || '';
+  const centuries = searchParams.getAll('centuries') || [];
+
+  function clearFilters() {
+    return ({
+      centuries: null,
+      sex: null,
+      query: null,
+    });
+  }
+
   return (
     <nav className="panel">
       <p className="panel-heading">Filters</p>
 
       <p className="panel-tabs" data-cy="SexFilter">
-        <a className="is-active" href="#/people">All</a>
-        <a className="" href="#/people?sex=m">Male</a>
-        <a className="" href="#/people?sex=f">Female</a>
+        <SearchLink
+          data-cy="century"
+          className={cn({ 'is-active': sex.includes('all') })}
+          params={{
```

Comment:
it adds strange params to url:
![image](https://github.com/mate-academy/react_people-table-advanced/assets/86923089/99778bc0-f9b7-47e3-9bc7-d2d94732c014)

it should just set centuries search param to `null` - i.e. similar as you do in the `clearFilters` only for centuries